src/dbspro/cli/run.py

```diff
@@ -0,0 +1,88 @@
+"""
+Run DPS-Pro pipeline
+"""
+
+import logging
+import os
+from pathlib import Path
+import sys
+import pkg_resources
+from snakemake import snakemake
+
+from dbspro.utils import available_cpu_count
+
+logger = logging.getLogger(__name__)
+
+accepted_file = 'reads.fastq.gz'
+accepted_file_ext = '.fastq.gz'
+
+
+def main(args):
+
+    cwd = Path(os.getcwd())
+
+    # Check if path to output directory is absolute or make it so.
+    if not os.path.isabs(args.directory):
+        args.directory = cwd / args.directory
```

Comment:
Since the files are checked for being absolute I don't think this is necessary and the `os.mkdir()` command should work without it I believe.

Reply:
Yeah but I also use that directory path to create symlinks for any given input files. In this case I am pretty certain that absolute paths are required.